pkgs/os-specific/linux/pam_gnupg/default.nix

```diff
@@ -0,0 +1,30 @@
+{ stdenv, fetchgit, autoreconfHook, gnupg, pam } :
+
+stdenv.mkDerivation rec {
+  pname = "pam_gnupg";
+  version = "unstable-2019-12-06";
+
+  src = fetchgit {
+    url = https://github.com/cruegge/pam-gnupg;
+    rev = "fbd75b720877e4cf94e852ce7e2b811feb330bb5";
+    sha256 = "0kqn6xb85jfmhvvbd2lasnci46p2pcwy0wq233za9h7xwfr49f7d";
+  };
+
+  nativeBuildInputs = [ autoreconfHook ];
+  buildInputs = [ gnupg pam ];
+
+  preAutoreconf = ''
+    mkdir m4
+  '';
+
+  configurePhase = ''
+    ./configure --prefix=$out --with-moduledir=$out/lib/security
+  '';
```

Comment:
replace with:

```
configureFlags = [ "--with-moduledir=$out/lib/security" ];
```

Reply:
I tried this originally, but the build failed because the shell somehow greedily expanded "$o" to an empty string and made moduledir "ut/lib/security". The configurePhase redefinition didn't cause the same problem. It turns out that using quoted braces around "out" when using configureFlags also solves the problem, so the latest commit adopts the correct form.